README.md

```diff
@@ -1,9 +1,9 @@
 # Grails MongoDB Database Migration Plugin
-Latest Version (0.2.4)
+Latest Version (0.2.5)
 
 ## Overview 
 
-This MongoDB database migration plugin helps to manage database changes while developing Grails 3.2.x Applications with MongoDB.
+This MongoDB database migration plugin helps to manage database changes while developing Grails 3.3.x and 3.2.x Applications with MongoDB.
```

Comment:
Please mention the versions supported for respective Grails versions.

Reply:
It supports in all 3.2.x and 3.3.x so not better to explicitly mention grails version.